source/dofs/dof_handler_policy.cc

```diff
@@ -851,17 +873,30 @@ namespace internal
 
                                     --unique_sets_of_dofs;
 
-                                    for (unsigned int j = 0;
-                                         j < dof_handler.get_fe(fe_index_1)
-                                               .dofs_per_line;
+                                    // determine which one of both finite
+                                    // elements is the dominating one.
+                                    std::set<unsigned int> fe_indices;
+                                    fe_indices.insert(fe_index_1);
+                                    fe_indices.insert(fe_index_2);
+
+                                    const unsigned int dominating_fe_index =
```

Comment:
The error for the test `fe/fe_enriched_color_08` is that `find_face_dominating_fe_in_subset()` returns an `invalid_unsigned_int` for `fe_index=1` and `fe_index=2`.

This should not happen. I'll investigate.

Reply:
I guess applying the domination algorithm at this point is simply overkill. Since we already checked beforehand that both finite elements are equivalent in terms of amount of dofs, we should rather choose the lower index of both.

I'll remove this part again.

Reply:
Well, I ended up keeping the FE domination logic in the code. If it fails, we now pick the lower index as a fallback solution. `fe/fe_enriched_color_08` is happy about that.